kitchensink/extensions.py

```diff
@@ -1,12 +1,44 @@
-from flask_fulfil import Fulfil
+import flask
+import os
+
+from fulfil_client.oauth import Session
+from fulfil_client import Client, BearerAuth, ClientError
 from flask_babel import Babel
 from flask_login import LoginManager
 from flask_debugtoolbar import DebugToolbarExtension
-#from flask.ext.celery import Celery
+from werkzeug.local import LocalProxy
+from .settings import Config
+# from flask.ext.celery import Celery
+
+# Setup fulfil session
+Session.setup(
+    os.environ['FULFIL_APP_ID'], os.environ['FULFIL_APP_SECRET']
+)
 
-#celery = Celery()
-fulfil = Fulfil()
+# celery = Celery()
 babel = Babel()
 toolbar = DebugToolbarExtension()
 login_manager = LoginManager()
-login_manager.login_view = "user.login"
\ No newline at end of file
+login_manager.login_view = "user.login"
+login_manager.login_message = None
```

Comment:
add comment why None.
`login page is now converted to redirect, this message will only be visible when user comes back after login.`